src/rf/apps/workers/status_updates.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+
+from datetime import datetime
+
+from apps.core.models import LayerImage, Layer
+import apps.core.enums as enums
+
+
+TIMEOUT_SECONDS = (60 * 10)  # 10 Minutes.
+
+JOB_VALIDATE = [
+    'ObjectCreated:CompleteMultipartUpload',
+    'ObjectCreated:Put',
+    'ObjectCreated:Post'
+]
+JOB_HANDOFF = 'emr_handoff'
+JOB_TIMEOUT = 'timeout'
+
+TIMEOUT_DELAY = 60
+
+
+def mark_image_valid(s3_uuid):
+    try:
+        image = LayerImage.objects.get(s3_uuid=s3_uuid)
+        image.status = enums.STATUS_VALID
+        image.save()
+        return True
+    except (ValueError, LayerImage.DoesNotExist):
```

Comment:
Consider passing a layer object instead of `s3_uuid` for these methods so we don't have to repeat this try/catch throughout.